tinasharma/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
adding .DS_Store would be good. Also, shouldn't there be a 'db' line added, if your --dbpath is in the /db folder?